README.md

```diff
@@ -32,6 +32,18 @@ simply dropped on read.  The maximum message length depends on the particular
 browser you connect with, but in my testing currently it is slightly smaller
 than 1200 bytes.
 
+## Running the example
+
+In ```./examples/```:
+
+```
+$cargo run --example echo_server -- --data 127.0.0.1:42424 --http 127.0.0.1:8080 --public 127.0.0.1:42424
+```
+
+Following that check out the example http service and see if you need to update it:
```

Comment:
I don't think you need to change anything at all still, this is too much information.